server/src/test/java/org/elasticsearch/health/node/LocalHealthMonitorTests.java

```diff
@@ -63,7 +62,7 @@ public class LocalHealthMonitorTests extends ESTestCase {
 
     @BeforeClass
     public static void setUpThreadPool() {
-        threadPool = new TestThreadPool(HealthNodeExecutorTests.class.getSimpleName());
+        threadPool = new TestThreadPool(LocalHealthMonitorTests.class.getSimpleName());
```

Comment:
👏

Reply:
Credits to Intellij refactoring functionality 😉.